css/document.css

```diff
@@ -263,10 +263,14 @@ The leader works by filling the li with dots, and arranging for the <a> and a::a
   padding-right: 0.25rem;
 }
 
+[data-id="content"] [data-id="toc"] {
```

Comment:
Let's do one change at a time, otherwise it will be harder to review, could you please revert these changes?

Reply:
Yes. Much agreed